src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIService.kt

```diff
@@ -55,6 +55,7 @@ class CommunityAPIService(
           }
           .subscribe()
       }
+      else -> {}
```

Comment:
Is this serving any purpose?

Reply:
oh it's a compiler warning i guess i fixed on a whim. not really is the answer